crates/trie/sparse/src/trie.rs

```diff
@@ -15,6 +15,15 @@ use reth_trie_common::{
 use smallvec::SmallVec;
 use std::{borrow::Cow, fmt};
 
+/// Struct for passing around `hash_mask` and `tree_mask`
+#[derive(Debug)]
+pub struct TrieMasks {
```

Comment:
let's add a helper method `TrieMasks::none()` so that we can use it instead of manually typing `TrieMasks { hash_mask: None, tree_mask: None }`